src/test/java/Tests/TestAgreement.java

```diff
@@ -0,0 +1,20 @@
+package Tests;
+
+
+import mainpack.module.Agreement;
+import mainpack.module.subject.AbstractSubject;
+import mainpack.module.subject.Apartment;
+import org.junit.jupiter.api.Assertions;
+import org.testng.annotations.Test;
+
+
+public class TestAgreement {
+    @Test
+    void testCountSubject(){
```

Comment:
Unit test should be named after tested method
In this test you are testing "getIncome" method, but you naming this test "testCountSubject", and not "testGetIncome"